lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -30,6 +30,7 @@
 author: Alan Loi (@loia)
 requirements:
   - "boto >= 2.37.0"
+  - "boto3 >= 1.4.4 (for tagging)"
```

Comment:
This seems unlikely - is 1.4.4 really the first version of boto3 capable of tagging dynamo DB tables?